state/upgrade/types.go

```diff
@@ -54,7 +54,11 @@ func (a OldAddress27) Upgrade(lookup map[string]string) (OldAddress27, error) {
 			logger.Warningf("not converting address %q with empty space name and ID %q", a.Value, a.SpaceID)
 		}
 	} else {
-		newID, ok := lookup[a.SpaceName]
```

Comment:
are comments from that upgrade function still correct? `// convert accepts an address a...`

Reply:
I've updated it with the method name.